backend-poll-generator/backend/workers/formol_eventstream.js

```diff
@@ -18,3 +18,18 @@ const tasks = require('../tasks');
 // );
 
 // module.exports = worker;
+const worker = new Worker("queueName",
+    {connection:redisConn},
+    async job =>{
+        if(job.ID === "12test34"){
+            worker.on("completed", (jobId)=>{
```

Comment:
This block and the block in L29 are not needed. You can that empty for now or with a cute little message.

Reply:
ok understood !!!